static/js/index.js

```diff
@@ -1 +1,32 @@
 // Your code starts here
+$(document).ready(function() {
+
+    $.ajax({
+        method: "GET",
+        url: "api/pokemon",
+        dataType: "json",
+        success(data){
+            //reference a class in the html with $(".class") which is how jquery does it
+            //append adds some stuff to the end of whatever div you've referenced
+            //creating strings with `` allows you to use string interpolation in jquery
+            //to actually add in the interpolated data, use ${data}
+            //you reference json items the same way you'd reference a multi-dimensional array i.e. data[num]["name"]
+            for (i=0; i < 151; i++) { 
```

Comment:
You should use a variable here instead of hardcoding the length so if we add more pokemon to the list it won't break